docs/src/visualization.md

```diff
@@ -85,6 +85,15 @@ that can be opened with any image viewer:
 
 !["solution_000040_scalar_resized"](https://github.com/trixi-framework/Trixi2Img.jl/raw/master/docs/src/assets/solution_000040_scalar_resized.png)
 
+To visualize 3D data generated by Trixi.jl, a 2D slice must be extracted.
+A slice can only lie on a plane orthogonal to an axis.
+The slice plane is defined by the axis to which it is orthogonal and the axis intercept.
+
+For example, to slice 3D data at the axis `z = 0.5`, execute
+```julia
+julia> trixi2img(joinpath("out", "solution_000000.h5"), output_directory="out", grid_lines=true, slice_axis=:z, slice_axis_intercept=0.5)
+```
+
```

Comment:
Can you please apply the same changes as in https://github.com/trixi-framework/Trixi2Img.jl/pull/20?